crates/flv/src/script.rs

```diff
@@ -40,17 +251,65 @@ mod tests {
 
     #[test]
     fn test_script_data() {
-        let mut reader = io::Cursor::new(Bytes::from_static(&[
-            0x02, // String marker
-            0x00, 0x0A, // Length (10 bytes)
-            b'o', b'n', b'M', b'e', b't', b'a', b'D', b'a', b't', b'a', // "onMetaData"
-            0x05, // null marker
-            0x05, // null marker
-        ]));
+        let width = 1280.0f64.to_be_bytes();
+        let data = [
+            Amf0Marker::String as u8,
+            0, // Length (10 bytes)
+            10,
+            b'o', // "onMetaData"
+            b'n',
+            b'M',
+            b'e',
+            b't',
+            b'a',
+            b'D',
+            b'a',
+            b't',
+            b'a',
+            Amf0Marker::Object as u8,
+            0, // Length (5 bytes)
+            5,
+            b'w', // "width"
+            b'i',
+            b'd',
+            b't',
+            b'h',
+            Amf0Marker::Number as u8,
+            width[0],
+            width[1],
+            width[2],
+            width[3],
+            width[4],
+            width[5],
+            width[6],
+            width[7],
+            0,
+            0,
+            Amf0Marker::ObjectEnd as u8,
+        ];
```

Comment:
Can you make this not reformat `#[rustfmt::skip]`
and then manually format it to make sense